modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java

```diff
@@ -26,9 +26,9 @@
 public class CodegenParameter {
     public boolean isFormParam, isQueryParam, isPathParam, isHeaderParam,
             isCookieParam, isBodyParam, hasMore, isContainer,
-            secondaryParam, isCollectionFormatMulti, isPrimitiveType, isModel;
+            secondaryParam, isCollectionFormatMulti, isPrimitiveType, isModel, shouldExplode;
```

Comment:
@ackintosh thanks for the PR. What about using `isExplode` to follow the naming convention of other boolean flags?

cc @OpenAPITools/generator-core-team

Reply:
Please update the following functions to include the new property:

https://github.com/OpenAPITools/openapi-generator/blob/df8d9d1188a117ec71af88790036c07d26179de6/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java#L100

https://github.com/OpenAPITools/openapi-generator/blob/df8d9d1188a117ec71af88790036c07d26179de6/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java#L181

https://github.com/OpenAPITools/openapi-generator/blob/df8d9d1188a117ec71af88790036c07d26179de6/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java#L254

https://github.com/OpenAPITools/openapi-generator/blob/df8d9d1188a117ec71af88790036c07d26179de6/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenParameter.java#L323

Reply:
Updated the comment according to the renaming.
https://github.com/OpenAPITools/openapi-generator/pull/4042#issuecomment-541379088